cypress/component/SignIn/sign_in.spec.js

```diff
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import { mount } from 'cypress/react';
-import SignIn from '../../../src/components/SignIn';
+import SignInButton from '../../../src/components/SignInButton';
 import { Config } from '../../../src/libs/config';
 
 const signInText = 'Sign-in';
```

Comment:
This text is no longer accurate - the first test should pass with 
```suggestion
const signInText = 'Sign In';
```